learners/nonuniform_quantization/learner.py

```diff
@@ -272,7 +283,10 @@ def __build_train(self):
       # define the ops
       with tf.control_dependencies(self.update_ops):
         self.ops['train'] = optimizer.apply_gradients(grads, global_step=self.ft_step)
-
+        if FLAGS.nuql_opt_mode != 'weights' and FLAGS.nuql_enbl_rl_agent == True:
```

Comment:
Please use `... and FLAGS.nuql_enbl_rl_agent` instead of `... and FLAGS.nuql_enbl_rl_agent == True`.